.github/workflows/build_task2.yml

```diff
@@ -5,7 +5,7 @@ on: [workflow_dispatch]
 jobs:
   build:
 
-    runs-on: microsoft-latest
+    runs-on: ubuntu-latest
```

Comment:
will it generate EXE for Windows

Reply:
-- no  i get  an error when trying to build the project  in ci 
  C:\Program Files\dotnet\sdk\6.0.402\Microsoft.Common.CurrentVersion.targets(1220,5): error MSB3644: The reference assemblies for .NETFramework,Version=v3.0 were not found. To resolve this, install the Developer Pack (SDK/Targeting Pack) for this framework version or retarget your application. You can download .NET Framework Developer Packs at https://aka.ms/msbuild/developerpacks

Reply:
i did try something new and now it is generating a exe file, please check it 
https://github.com/Bozhyk-Oleh/csdt2223ki46bozhyk02/actions/runs/3272637944/jobs/5383887733

Reply:
Good, but how I can download EXE?

Reply:
DONE!
https://github.com/Bozhyk-Oleh/csdt2223ki46bozhyk02/actions/runs/3276904833